src/firefly/js/visualize/ui/ImageSelectPanelProp.js

```diff
@@ -28,6 +28,35 @@ export const initPanelCatalogs = [
         'range': {'min': 1, 'max': 12.5, 'unit': 'deg'},
         'size': 5
     },
+    /*
```

Comment:
I don't see atlas showing up in the ImageSelectPanel is this work completed?

Reply:
no, i disabled it because it need more work to show it on that panel and it will be changed in IRSAViewer by the new layout. If you want, you can see it in "Test searches" for single image (see comment on the AtlasRequestList.js code).